### DISCUSSION TITLE
Border Gradient

### DISCUSSION THREAD

**Discussion by imjohnlouie04** [tailwindlabs/tailwindcss#4707] [Ideas]
It will be cool to add

can be like `border-g-to border-g-via border-g-from `
or like `border-gradient` as parent class followed by https://tailwindcss.com/docs/gradient-color-stops 

https://codepen.io/johnlouie04/pen/mdWojar

**Comment by deadcoder0904** [comment]
Yes, would love this as well. I have recently been trying to add top-border to make my text-based site (as per Refactoring UI suggestions) look a bit more professional & I couldn't add `border-gradient` to it.

I think with JIT mode it could be possible as with all the new features in Tailwind v2.2 :)
